src/app/cube/content-pages/contribute-page/contribute-page.component.scss

```diff
@@ -0,0 +1,79 @@
+@import 'vars.scss';
+h1 {
+    text-align: center;
+    padding-top: 50px;
+    padding-bottom: 25px;
+    color: $light-blue;
+    position: relative;
+    font-size: 27pt;
```

Comment:
We should use the font size variables in the vars.scss file.

Reply:
Agreed

Reply:
You can access them the same way you access the color variables.